src/transactions/CategoryFilter.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import { CATEGORIES } from '../data';
+
```

Comment:
We need a JSDoc comment here that documents the CategoryFilter component. Think of what another developer would need to know about CategoryFilter if they needed to use it, but couldn't read your code. You can see examples of this in TransactionForm, TransactionTable, etc.